packages/schemas/src/foundations/jsonb-types.ts

```diff
@@ -72,6 +73,23 @@ export const userLogPayloadGuard = z.object({
 
 export type UserLogPayload = z.infer<typeof userLogPayloadGuard>;
 
+/**
+ * Logs
+ */
+
+export const logPayloadGuard = z.object({
+  ip: z.string().optional(),
+  userAgent: z.string().optional(),
+  applicationId: z.string().optional(),
+  connectorId: z.string().optional(),
+  userId: z.string().optional(),
+  request: z.object({ body: z.object({}).optional() }).optional(),
+  response: z.object({ body: z.object({}).optional() }).optional(),
+  // TODO: Add necessary fields when needed (LOG-2193)
```

Comment:
is these fields and comment necessary now? we can remove until needed

Reply:
don't add more to-do comments unless we have to

Reply:
Removed. PTAL